subgraph/schema.graphql

```diff
@@ -0,0 +1,100 @@
+type MACI @entity {
+  id: ID!
+  owner: Bytes! # address
+  stateTreeDepth: Int! # uint8
+  updatedAt: BigInt!
+  blockNumber: BigInt!
+  txHash: Bytes!
+
+  "state"
+  numRegistrar: BigInt!
+  numPoll: BigInt!
+  numVotes: BigInt!
```

Comment:
`numSignUps`? I believe votes would tracked on `Poll`, not `MAC`

Reply:
thanks, has been changed to `numSignUps`

Reply:
Has it? I'm still seeing `numVotes`

Reply:
ohh! my bad, I replied before committing the changes